protoc-gen-go/grpc/grpc.go

```diff
@@ -130,8 +130,8 @@ func (g *grpc) GenerateImports(file *generator.FileDescriptor) {
 		return
 	}
 	g.P("import (")
-	g.P(contextPkg, " ", strconv.Quote(path.Join(g.gen.ImportPrefix, contextPkgPath)))
-	g.P(grpcPkg, " ", strconv.Quote(path.Join(g.gen.ImportPrefix, grpcPkgPath)))
+	g.P(contextPkg, " ", strconv.Quote(path.Join(string(g.gen.ImportPrefix), contextPkgPath)))
```

Comment:
Could just do:
```go
generator.GoImportPath(path.Join(string(g.gen.ImportPrefix), context.PkgPath))
```
instead of:
```go
strconv.Quote(path.Join(string(g.gen.ImportPrefix), contextPkgPath))
```
to be type correct.

Perhaps even make Join a method of `ImportPrefix`:
```go
g.gen.ImportPrefix.Join(context.PkgPath)
```

Reply:
Done.

Didn't make Join a method, because ImportPrefix is terrible and doesn't need more support.